app/helpers/application_helper.rb

```diff
@@ -515,4 +515,14 @@ def marketing_mat_count
   def round_down_to_nearest_ten(val)
     (val / 10) * 10
   end
+
+  def admin_link(path, to: 'Edit', tag: nil, classes: nil)
+    if current_user&.admin?
+      link = link_to to, path,
+                  class: "badge badge-light font-weight-normal border align-text-top #{classes}",
```

Comment:
I'm not sure about using badges rather than proper buttons, I'm not sure its immediately obvious that a badge style link means you can edit something. Most other places we're just using buttons I think. Can you change?